delivery/router.go

```diff
@@ -0,0 +1,13 @@
+package delivery
+
+import (
+	"lolidelgado/github-users/controller"
+
+	"github.com/gorilla/mux"
+)
+
+func Setup(githubUserHandler *controller.GithubUser, router *mux.Router) {
+
+	router.HandleFunc("/github-users", githubUserHandler.GetGithubUsers)
+	router.HandleFunc("/github-users/{id}", githubUserHandler.GetGithubUserById)
+}
```

Comment:
A pro-tip, mux contains a method named `Methods` that allows you to specify which HTTP methods you are going to support on your `handleFunc` for the given route. No need to validate in the controller if that's a GET request.

Also, it is important to always comment your functions, methods, structs, variables, constants, etc., as for you it makes sense what it does, but for another person that might want to import and use your package, it gives an idea what this element does, and also it works as a way to document your code; this is the standard in Go.

You can give it a look into https://go.dev/doc/comment for a more in-deep explanation. 
```suggestion
// Setup adds the routes to the given router
func Setup(githubUserHandler *controller.GithubUser, router *mux.Router) {
	router.HandleFunc("/github-users", githubUserHandler.GetGithubUsers).Methods(http.MethodGet)
	router.HandleFunc("/github-users/{id}", githubUserHandler.GetGithubUserById).Methods(http.MethodGet)
}
```